src/details/ArborX_DetailsMortonCode.hpp

```diff
@@ -81,6 +96,24 @@ unsigned int morton3D(float x, float y, float z)
          expandBitsBy2((unsigned)z);
 }
 
+KOKKOS_INLINE_FUNCTION
+unsigned long long morton3D64bit(float x, float y, float z)
```

Comment:
I think the `3D` part of the name is unnecessary.

Reply:
You could do 64 bit Morton codes in 2D, too, where full 32 bits of both x and y are used. May implement at some point, because current 2D 32-bit resolution is just 16 times better than 3D 32-bit.

Reply:
Sure but the functions take 2 and 3 args resp so it is unnecessary

Reply:
I'm not opposed to it, but then would also change the existing names. So it would be
```
unsigned morton32(float x, float y);
unsigned morton32(float x, float y, float z);
unsigned long long morton64(float x, float y);
unsigned long long morton64(float x, float y, float z);
```
It would also be good if we could come up with a naming that would not need to be changed in the future if we decide to implement extended Morton codes.